src/mpid/ch4/netmod/ofi/ofi_pre.h

```diff
@@ -81,10 +83,12 @@ typedef struct MPIDI_OFI_am_header_t {
     uint64_t handler_id:MPIDI_OFI_AM_HANDLER_ID_BITS;
     uint64_t am_type:MPIDI_OFI_AM_TYPE_BITS;
     uint64_t am_hdr_sz:MPIDI_OFI_AM_HDR_SZ_BITS;
-    uint64_t data_sz:MPIDI_OFI_AM_DATA_SZ_BITS;
-    int32_t seg_sz;             /* size of current pipeline segment */
-    uint16_t seqno;             /* Sequence number of this message.
-                                 * Number is unique to (fi_src_addr, fi_dest_addr) pair. */
+    uint64_t payload_sz:MPIDI_OFI_AM_PAYLOAD_SZ_BITS;   /* data size on this OFI message. This
+                                                         * could be the size of a pipeline segment
+                                                         * */
+    uint64_t seqno:MPIDI_OFI_AM_SEQ_NO_BITS;    /* Sequence number of this message.
```

Comment:
promoting this to 64-bits then limiting to 16 with a bitfield is kind of weird to include in this change

Reply:
Will fix.